Makefile

```diff
@@ -23,12 +23,19 @@ release-local: prepare
 	goreleaser release --rm-dist --snapshot --skip-publish -f ./build/ci/goreleaser.yml
 
 TEMPLATES_API_BUILD_TARGET_PATH ?= "./target/templates-api/server"
-build-templates-api: prepare
+build-templates-api: generate-templates-api
 	CGO_ENABLED=0 go build -v -mod mod -ldflags "-s -w" -o $(TEMPLATES_API_BUILD_TARGET_PATH) ./cmd/templates-api
 
-run-templates-api: prepare
+run-templates-api: generate-templates-api
 	air -c ./provisioning/templates-api/dev/.air-templates-api.toml
 
+BUFFER_API_BUILD_TARGET_PATH ?= "./target/buffer-api/server"
```

Comment:
Přidaný buffer api targets. Plus jedna maličkost, asi není potřeba spouštět celý `prepare` pro obě API, tak se spouští jen generate pro to jedno API, které se builduje/spouští.

Reply:
:+1: